ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -136,6 +137,10 @@ const BugDetailsView = props => {
       header={
         <React.Fragment>
           <Row>
+          <Helmet>
+            <meta charSet="utf-8" />
```

Comment:
Why did you add a meta tag here? I don't think this would need to be in the row.

Reply:
I’ll make these changes! Thanks for reviewing. I kept it there but I’ll remove it!